assets/enx-redirect/report/index.html

```diff
@@ -8,18 +8,25 @@
   {{template "report/header" .}}
 </head>
 
-<body class="tab-content">
+<body class="my-4 g-4">
   <main role="main" class="container">
     {{if $currentRealm.AgencyImage}}
     <div id="logo" style="background-color: {{$currentRealm.AgencyBackgroundColor}};" >
-      <img src="{{$currentRealm.AgencyImage}}" style="width: 80vw; max-width: 720px;" />
+      <img src="{{$currentRealm.AgencyImage}}" style="width:80vw; max-width:720px; content-visibility:auto;"
+        loading="lazy" decoding="async"
```

Comment:
This allows the page to continue rendering while this image is loading. This is important for folks on really slow connections because, without it, they have to wait for the entire image to load before seeing the form.